docs/BINDING_PROCESS_DIAGRAM.md

```diff
@@ -0,0 +1,148 @@
+# RAMSES RF Binding Process Diagram
+
+## Overview
+This diagram shows the binding finite state machine (FSM) for RAMSES RF devices, illustrating the complete process from initial state to binding completion.
+
+## State Machine Diagram
+
+```
+┌─────────────────────┐    initiate_binding_process()      ┌─────────────────────────────┐
```

Comment:
For now, this is nice. In UML/mermaid use the [state diagram](https://mermaid.js.org/syntax/stateDiagram.html) type